### PR TITLE
Fix/make it easier to get first aid 1

### LINKED ISSUE #24912: Traning First aid 1 without books is tedious.
The only real way to train this skill without books is to make bandages ( which dont train the skill ) and apply them one by one. Each application is 1% so it takes 100 bandages and 100 applications of said bandages to a hurt part before you scan skill up to 1. 1 is needed to heal broken limbs since they wont mend without a splint. 

This could be remedied by having making bandages skill use first aid instead of tailoring.

### PR DESCRIPTION
**Pull request by FeepingCreature**
#### Summary
```SUMMARY: Balance "Give 2.5x boost for training first aid to level 1"```

#### Purpose of change
First aid level 1 is where you get splints. Without splints, broken bones never heal naturally. (Bug report?) When playing a wilderness start at first aid 0, it can take a lot of bandages to train yourself up to the required level.

#### Describe the solution
This patch gives a 2.5x boost to healing xp gain at level 0, but keeps the 9% cap in place.

#### Describe alternatives you've considered
Giving a flat 2.5x boost after the 9xp cap may lead to unpredictably overpowered xp gain.

#### Additional context
This is not unrealistic, since at first aid level 0 there shouldn't be that big a difference in skill gain, because you probably won't be taking the nature of the wound into account yet. With the patch, you can more easily hit your "learning ceiling" per healing.
Fixes #24912.

### CODE REVIEW

**Comment by nexusmrsep** [comment]
Leaflets about first aid in firstaid kits were made for that purpose.

**Comment by FeepingCreature** [comment]
How do I find that in the wilderness?

**Comment by nexusmrsep** [comment]
I don't deny that. What are other ways of learning first aid? I at least know of one - dissecting corpses. That also might be as tedious in wilderness without a fine cutting tool at hand. First aid is one of the skills that has not enough ways to train in general.

**Comment by Inglonias** [comment]
Another solution would be to make splints available at first aid 0. They are a pretty essential item if the worst happens.

**Comment by kevingranade** [comment]
Let me rephrase your "problem statement".

> When playing a wilderness start *with no first aid knowledge*, you run the risk of injuring yourself and being unable to properly treat yourself.  Also, it is difficult to learn first aid via trial and error when alone in the wilderness.

This is working as intended, you're choosing a challenge start, either tough it out or spend some points on first aid skill.

**Comment by FeepingCreature** [comment]
> Let me rephrase your "problem statement".
> 
> > When playing a wilderness start _with no first aid knowledge_, you run the risk of injuring yourself and being unable to properly treat yourself.  Also, it is difficult to learn first aid via trial and error when alone in the wilderness.
> 
> This is working as intended, you're choosing a challenge start, either tough it out or spend some points on first aid skill.

Actually, to be really specific - that's not the issue. The issue is that there _is_ no trial and error, because there's _nothing I can try._ If splints had a very high failure chance at level 0, that would be trial and error.

**Comment by l29ah** [comment]
I've got first aid 1 in no time by applying bandages: you get skill points for them healing you as they stay on you. It would still be tricky to get the rags in the wilderness.

**Comment by FeepingCreature** [comment]
!

Oh.

Okay.

I was going off bad information in that case, I thought it was still in the 1% range as listed in the op.

It's still kind of weird that I have to injure myself so I can get experience in treating myself so I can work up to *splints*, which have nothing to do with wound healing.
